Sources/TGCardViewController/cards/TGTableCard.swift

```diff
@@ -155,17 +155,17 @@ open class TGTableCard: TGCard {
         return
     }
     
-    if scrollCardView.safeAreaInsets.bottom == 0 {
-      // For devices with physical Home button, we need to add some extra padding
-      // from the bottom, otherwise, the last row of the table view will be
-      // partially hidden below the screen.
-      embeddedScrollView.contentInset.bottom = scrollCardView.safeAreaInsets.bottom + 16
-      embeddedScrollView.verticalScrollIndicatorInsets.bottom = scrollCardView.safeAreaInsets.bottom + 16
-    } else {
-      embeddedScrollView.contentInset.bottom = scrollCardView.safeAreaInsets.bottom
-      embeddedScrollView.verticalScrollIndicatorInsets.bottom = scrollCardView.safeAreaInsets.bottom
+    var adjustment: CGFloat = 0
```

Comment:
Can you add a comment what this does and why?

By the way, this should have nothing to do with the safe area insets of `TGMapVC` as those insets are contained to that VC and the cards are elsewhere in the view hierarchy, so this likely was broken before.